styles.css

```diff
@@ -405,8 +450,220 @@ a.active {
+}
+
+.ct-img-txt-title {
+  font-weight: 400;
+}
 
+.ct-img-txt-main {
+  font-weight: bold;
+  font-size: 22px;
+  text-decoration: none;
+}
+
+.ct-img-txt-sub {
+  color: #888;
+}
+
+.subtitle-left {
+  margin: 15px;
+}
+
+.ct-sub1 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/1280px-20th_Anniversary_Schwab_Foundation_Gala_Dinner_44887783681-e1611132397581.jpg") no-repeat;
+}
+
+.ct-sub2 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/header-image-bernie-sitting-meme-analysis.png");
+}
+
+.ct-sub3 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/11/Apple-AR-glasses-headset.jpg");
+}
+
+.ct-sub4 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/1-copy-52.jpg");
+}
+
+.ct-sub5 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/09/Untitled-design-13.jpg");
+}
+
+.ct-sub6 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/10/google-logo-podium.png");
+}
+
+.ct-sub7 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/dev-leadership-gq.png");
+}
+
+.ct-sub8 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/10/hummer-ev-future-car-love-hate-next-gmc-gm-ultium.jpg");
+}
+
+.ct-sub9 {
+  background: url("https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/bloodlines.jpg");
+}
+
+.subtitle-right {
+  background-size: 100% 100%;
+  background-position: 40%;
+  flex: 0 0 96px;
+  width: 96px;
+  height: 96px;
+}
+
+.subtitle-txt {
+  font-weight: bold;
+  margin-top: 3px;
+}
+
+.categories-subtitle-img {
+  display: flex;
+  flex: 1 1 80px;
+  background-color: #ff442214;
+  color: #f42;
+}
+
+.categories-links {
+  flex: 1 1 300px;
+}
+
+.categories-ul {
+  display: flex;
+  flex-direction: column;
+  list-style-type: circle;
+  list-style-position: inside;
+}
+
+.categories-li {
+  padding: 10px 5px 10px 0;
+  border-bottom: 0.5px solid rgba(211, 181, 181, 0.658);
+}
+
+/* Latest deals */
+
+.latest-deals-container {
+  margin: 0 20px 20px 20px;
+}
+
+.ld-grid {
+  margin-left: 10px;
+}
+
+.ld-title {
+  color: #f42;
+  font-size: 23px;
+  font-weight: bolder;
+  margin: 9px;
+}
+
+.ld-title i {
+  font-size: 15px;
+}
+
+.ld-content {
+  display: flex;
+  margin-top: 10px;
+}
+
+.ld-txt {
+  margin-left: 10px;
+}
```

Comment:
## Aim to Stay `DRY`
- [ ] The lines of code on line `#628` & `#648` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant?